#!/usr/bin/env python3
# -*- coding: utf-8 -*-
# (c) ACE 

import os

class Config(object):
    # get a token from @BotFather
    pass
    """
    BOT_TOKEN = os.environ.get("BOT_TOKEN", "7069765759:AAGBgkSBQSEQeN_FMJ3u1WkCyW7pHO9tQYs")
    API_ID = int(os.environ["API_ID", 26209173]
    API_HASH = os.environ["API_HASH", "a4d86637bb1b8db35e92196493798279"]
    AUTH_USERS = "5954632422"""
# jisko bhi is repo ko deploy karne me error aa raha ho ( https://t.me/Stubborn1223 ) ye mere TG ka id ha message karke iska screen shot send kar dena me error solve kar dunga. 
